packages/haul-cli/src/commands/init.ts

```diff
@@ -307,11 +309,36 @@ async function addHaulScript(progress: Ora, cwd: string): Promise<string> {
   return scriptName;
 }
 
+async function getAvailableHaulPreset(
+  targetHaulPreset: string
+): Promise<string> {
+  // Stop searching on 0.59 - there's no preset below 0.59.
+  if (targetHaulPreset.includes('0.59')) {
+    return targetHaulPreset;
+  }
+
+  try {
+    await npmFetch(targetHaulPreset);
+    return targetHaulPreset;
+  } catch (error) {
+    if (error.statusCode === 404) {
```

Comment:
I think it would be nice to log a message about falling back to previous preset, wdyt?